README.md

```diff
@@ -195,274 +227,424 @@ print("t-critical value:")                  # Check the t-critical value
+    # Sample mean (blue circle with gray edge)
+    Line2D([0],[0],
+           marker="o",
+           color="w",
+           markerfacecolor="blue",
+           markeredgecolor="gray"
+          ),
+    # Confidence interval (gray vertical line)
+    Line2D([0],[0],
+           marker="|",
+           markersize=15,
+           color="w",
+           markeredgewidth=1.5,
+           markeredgecolor="gray"
+          ),
+    # Population mean (red horizontal line)
+    Line2D([0],[0], color="red")
+]
+
+ax.legend(
+    handles=legend_elements,
+    labels=["Sample Mean", "Sample Confidence Interval for Mean", "True Population Mean"],
+    loc="lower left", 
+    fontsize="large");
 ```
 
 
+```python
+# __SOLUTION__ 
+
+# Set up figure
+fig, ax = plt.subplots(figsize=(15,7))
 
+# Draw the means and confidence intervals for each sample
+ax.errorbar(x=np.arange(1, 21, 1),
+            y=sample_means,
+            yerr=[(upper-lower)/2 for upper, lower in intervals],
+            fmt='o',
+            color="gray",
+            markerfacecolor="blue")
 
-    <matplotlib.collections.LineCollection at 0x1a1927f400>
+# Draw the population mean as a horizontal line 
+ax.hlines(xmin=0, xmax=21,
+           y=population_ages.mean(), 
+           linewidth=2.0,
+           color="red")
 
+# Label plot
+ax.set_xlabel("Samples")
+ax.set_ylabel("Means")
+
+# Customize legend appearance
+legend_elements = [
+    # Sample mean (blue circle with gray edge)
+    Line2D([0],[0],
+           marker="o",
+           color="w",
+           markerfacecolor="blue",
+           markeredgecolor="gray"
+          ),
+    # Confidence interval (gray vertical line)
+    Line2D([0],[0],
+           marker="|",
+           markersize=15,
+           color="w",
+           markeredgewidth=1.5,
+           markeredgecolor="gray"
+          ),
+    # Population mean (red horizontal line)
+    Line2D([0],[0], color="red")
+]
+
+ax.legend(
+    handles=legend_elements,
+    labels=["Sample Mean", "Sample Confidence Interval for Mean", "True Population Mean"],
+    loc="lower left", 
+    fontsize="large");
+```
 
 
+![png](index_files/index_32_0.png)
 
-![png](index_files/index_29_1.png)
 
+### Interpreting Confidence Intervals of Repeated Samples
 
-Just like the last lab, all but one of the 95% confidence intervals overlap the red line marking the true mean. This is to be expected: since a 95% confidence interval captures the true mean 95% of the time, we'd expect our interval to miss the true mean 5% of the time.
+Just like the last lab, all but one of the 95% confidence intervals overlap the red line marking the true mean. This is to be expected: since a 95% confidence interval captures the true mean 95% of the time, we'd expect our interval to miss the true mean 5% of the time. It won't always turn out at exactly 1 out of 20 like this (just like flipping a coin twice won't always give you 1 heads and 1 tails), but this particular example turned out nicely.
```

Comment:
As per the discussion above at line 385, replace this paragraph with something like the following:

'''
Just like the last lab, all but one of the 95% confidence intervals overlap the red line marking the true mean. This is to be expected: since we're generating a 95% confidence interval, we would expect after repeatedly sampling and calculating confidence intervals that 95% of the confidence intervals calculated would contain the true population mean, while 5% do not.  

It won't always turn out at exactly 1 out of 20 like this (just like flipping a coin twice won't always give you 1 heads and 1 tails), but this particular example turned out nicely.
'''